webapp/src/components/Analysis/UtterancesTable.tsx

```diff
@@ -97,6 +98,7 @@ type Props = {
   filters: QueryFilterState;
   pagination: QueryPaginationState;
   pipeline: QueryPipelineState;
+  withoutPostprocessing: QueryPostProcessingState;
```

Comment:
I think we should name the "family" of query parameters `postprocessing`, matching its type, similarly to how `pagination` is of type `QueryPipelineState`.
Also, for the type, to match the rest of our code base, I think we should stay consistent with `Postprocessing` in one word (no capital in the middle).
```suggestion
  postprocessing: QueryPostprocessingState;
```

Reply:
More details: That would avoid confusions like you had yesterday about `withoutPostprocessing.withoutPostprocessing` looking ridiculous, now being `postprocessing.withoutPostprocessing`.
Also, we can imagine other postprocessing-related query parameters joining this family, for example a `postprocessing.postprocessingType` that switches between different types. A `withoutPostprocessing.postprocessingType` wouldn't make sense.

Reply:
makes sense

Reply:
Thank you for renaming the filter family.
Now, what about the second part of the suggestion?
> Also, for the type, to match the rest of our code base, I think we should stay consistent with `Postprocessing` in one word (no capital in the middle).

Reply:
Plus there are 2 or 3 other places where we see `PostProcessing`, that should change to `Postprocessing`.

Reply:
Thank you for renaming this one. I think some more are left though.